backend_new/src/dtos/jurisdictions/jurisdiction.dto.ts

```diff
@@ -33,44 +36,53 @@ export class Jurisdiction extends AbstractDTO {
     each: true,
   })
   @IsDefined({ groups: [ValidationsGroupsEnum.default] })
+  @ApiProperty()
   languages: LanguagesEnum[];
 
   @Expose()
   @ValidateNested({ groups: [ValidationsGroupsEnum.default] })
-  @Type(() => MultiselectQuestion)
+  @Type(() => IdDTO)
   @IsDefined({ groups: [ValidationsGroupsEnum.default] })
-  multiselectQuestions: MultiselectQuestion[];
+  @ApiProperty()
+  multiselectQuestions: IdDTO[];
```

Comment:
Did you mean to switch this to IdDTO? Shouldn't this still be MultiselectQuestion?

Reply:
Oh nevermind, that is how the existing backend is setup